docs/contributing/reporting-bugs.md

```diff
@@ -138,11 +138,11 @@ renamed or resolved.
   an actual, reproducible bug, anyone can help out by filing a GitHub issue to
   track it:
   - In some cases, especially if we're planning to fix the issue right away, the
-    issue description can be a quote from a message with a link to the
+    issue description simply quote and link to a message from the original
```

Comment:
How about: "In some cases, especially if we're planning to fix the issue right away, simply quote and link to a message from the discussion in the development community for the GitHub issue description -- no need to stress over making it perfect."

Reply:
Sure. I'll rewrite you idea so that "for the GitHub issue description" doesn't sit as a misplaced modifier: "...right away, the GitHub issue description can simply quote and link to..."